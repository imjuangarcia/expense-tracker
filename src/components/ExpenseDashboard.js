import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpensesSummary from "./ExpensesSummary";

const ExpenseDashboard = () => (
  <section>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
  </section>
);

export default ExpenseDashboard;
