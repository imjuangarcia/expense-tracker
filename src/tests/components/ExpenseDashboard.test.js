import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboard from "../../components/ExpenseDashboard";

test("Should render the Expense Dashboard component", () => {
  const wrapper = shallow(<ExpenseDashboard />);
  expect(wrapper).toMatchSnapshot();
});
