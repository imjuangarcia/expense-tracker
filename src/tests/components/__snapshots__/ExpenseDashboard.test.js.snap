// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Expense Dashboard component 1`] = `
<section>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</section>
`;
